clients/da-vinci-client/src/main/java/com/linkedin/davinci/blobtransfer/BlobSnapshotManager.java

```diff
@@ -204,6 +214,15 @@ private boolean isSnapshotStale(String topicName, int partitionId) {
         - snapshotTimestamps.get(topicName).get(partitionId) > snapshotRetentionTimeInMillis;
   }
 
+  /**
+   * Increase the count of hosts using the snapshot
+   */
+  public void increaseConcurrentUserCount(String topicName, int partitionId) {
```

Comment:
nit: don't need to expose as _public_, we can use default modifier for unit test.